src/components/stack-chart/index.js

```diff
@@ -142,6 +142,7 @@ class StackChartGraph extends React.PureComponent<Props> {
               interval,
               thread,
               stackTimingByDepth,
+              // $FlowFixMe Error introduced by upgrading to v0.96.0.
               updatePreviewSelection,
```

Comment:
it would be super cool to fix this specific error as it appears in several locations.

Reply:
This is a follow-up, due to ThunkActions. https://github.com/firefox-devtools/profiler/issues/1936